_ecosys/producers/GeoREST.md

```diff
@@ -2,6 +2,6 @@
 layout: default
 category: producers
 title: "GeoREST"
-link: "http://code.google.com/p/georest/"
+link: "https://code.google.com/p/georest/"
```

Comment:
I get an error for this url, but this one works: [https://code.google.com/archive/p/georest/](https://code.google.com/archive/p/georest/)